cypress/integration/english/tag/tag.spec.js

```diff
@@ -10,4 +16,13 @@ describe("Tag page", () => {
   it("should render", () => {
     cy.contains(selectors.tagName, "#FREECODECAMP");
   });
+
+  // To do: run tests against a tag with more total posts
+  it(`should show 7 posts on load`, () => {
+    getPostCards().should("have.length", 7);
+  });
```

Comment:
Question: will this fail if we publish another article with that tag? Should we do:

```suggestion
  it(`should show 7 posts on load`, () => {
    getPostCards().should("have.length.gte", 7);
  });
```

Reply:
Hey @nhcarrigan, thanks for your review and for pointing this out.

Right now the CI builds are sourced from a couple of JSON files, and aren't likely to change.

Eventually we'll move to a Dockerized Ghost setup for testing, and we'll probably end up adding/editing posts and tags for future tests, which could break tests like this.

It definitely wouldn't hurt to guard against any future changes now :+1: :+1: